### PR TITLE
add freertos-addons and option to set path to portable directory

### PR DESCRIPTION
**Pull request by johnathancn**

### CODE REVIEW

**Comment by johnathancn** [comment]
#35 #34

File: freertos-cargo-build/src/lib.rs [hunk 1]
```diff
@@ -61,6 +62,7 @@ impl Builder {
         let freertos_path = env::var(ENV_KEY_FREERTOS_SRC).unwrap_or_default();
         let freertos_config_path = env::var(ENV_KEY_FREERTOS_CONFIG).unwrap_or_default();
         let freertos_shim = env::var(ENV_KEY_FREERTOS_SHIM).unwrap_or_default();
+        let freertos_port_base = format!("{}{}", freertos_path, "portable");
```
Comment (schteve) [inline freertos-cargo-build/src/lib.rs:65]: Would prefer to use something like `Path::join` here to ensure the separator is correct in all cases.

**Comment by schteve** [comment]
Thanks for this, could you please point the submodule to the latest released tag from the other repo? I would prefer to follow releases for simplicity.

Also, based on our conversation the fix for #35 is probably more involved than just adding a define. Feel free to remove those changes for now if you want to merge sooner.

**Comment by johnathancn** [comment]
> Thanks for this, could you please point the submodule to the latest released tag from the other repo? I would prefer to follow releases for simplicity.
> 
> Also, based on our conversation the fix for #35 is probably more involved than just adding a define. Feel free to remove those changes for now if you want to merge sooner.

Updated the tag to v1.6.0 and removed the #define patch

**Comment by schteve** [comment]
Looks good to me. Thanks again!

**Pull request merged**